src/commands/classes/Contest.command.ts

```diff
@@ -260,108 +175,117 @@ export default class ContestCommand extends Command {
+        fs.writeFileSync(path.join(__dirname,this.CONTEST_DATA_FILE) ,JSON.stringify(contestData, null, 2),{ flag: 'w' });
 
-        console.log("Contest has been Ended");
-        return;
-      }).catch(e => {
-        console.error("Contest End Failed: "+e);
-        return;
-      });
+        contestChannel.send(announcement, {embed: mEmbed});
+      })
+      .catch(e => console.info(e));
     }
     else{
-      console.log("No contest to end.");
-      message.reply("There is not an active contest.");
+      message.reply('There is not an active contest.');
     }
     
     return;
   }
 
-  public helpAction(args: Array<string>, message: Message): void{
-    message.reply(this.help);
+  public helpAction(args: Array<string>, message: Message): void {
+    message.reply(this.HELP_MESSAGE);
     return;
   }
 
   // Clear Past Entrants data to add them back to applicant pool
   public reset(){
-    let contestData = JSON.parse(fs.readFileSync(path.join(__dirname, "../../../data/contestData.json")).toString());
-    console.log("Clearing Past Entrant records");
+    let contestData = JSON.parse(fs.readFileSync(path.join(__dirname, this.CONTEST_DATA_FILE)).toString());
+    console.log('Clearing Past Entrant records');
     for(let entry in contestData.pastEntries){
       delete contestData.pastEntries[entry];
     }
     try{
-      fs.writeFileSync(path.join(__dirname,"../../../data/contestData.json"), JSON.stringify(contestData, null, 2),{ flag: 'w' });
+      fs.writeFileSync(path.join(__dirname, this.CONTEST_DATA_FILE), JSON.stringify(contestData, null, 2),{ flag: 'w' });
     }
     catch(e){
-      console.error("Warning! Failure writing contest data to file after !contest reset!: "+e);
+      console.error('Warning! Failure writing contest data to file after !contest reset!: ' + e);
       return;
     }
     return;
   }
 
-  public action(args: Array<string>, message: Message): void{
+  public action(args: Array<string>, message: Message): void {
     
-    if(args[0] === undefined) { message.reply("Please enter Sub-command: ex '!Contest help'")}
+    if(args[0] === undefined) { message.reply("Please enter Sub-command: ex '!contest help'")}
     else
     {
       let func = args.shift().toLowerCase();
-      if(func === "help") { func = "helpAction";}
-      try{
+      if(func === 'help') { func = 'helpAction';}
+      try {
           this[func](args, message);
       }
       catch (Error) {
           if (Error.name === 'TypeError') { 
-            console.error("Invalid Command"); console.error(Error.name+": " + Error.message); 
-            message.reply("Invalid Sub-command. Please refer to !Contest help for list of Sub-commands and their use.");
+            message.reply('Invalid sub-command. Please use "!contest help" for list of sub-commands.');
           } 
-            else {console.error(Error.name+": " + Error.message); }
-          }
+          else {console.error(Error.name+': ' + Error.message); }
```

Comment:
Strings with variables in them should use the special tick mark format \`${<variable_name>}\`

There are a few strings around not following this. Please update them.